python/pyarrow/table.pxi

```diff
@@ -1226,7 +1230,7 @@ cdef class Table(_PandasConvertible):
         raise TypeError("Do not call Table's constructor directly, use one of "
                         "the `Table.from_*` functions instead.")
 
-    def to_string(self, show_metadata=False):
+    def to_string(self, show_metadata=False, preview_cols=0):
```

Comment:
1) Should we make all arguments keyword-only?
2) The new argument should be documented below

Reply:
:+1:

and added docstrings for all three `to_string` methods.